core/src/main/java/com/yahoo/ycsb/generator/FileGenerator.java

```diff
@@ -73,16 +73,12 @@ public String lastString()
 	/**
 	 * Reopen the file to reuse values.
 	 */
-	public synchronized void reloadFile()
+	public synchronized void reloadFile() throws IOException
```

Comment:
Are these errors caught somewhere else? Is there a reason that exceptions were just printed instead of being thrown previously?